jwst/tests_nightly/general/nirspec/test_nirspec_steps_single.py

```diff
@@ -146,7 +146,7 @@ def test_nirspec_nrs1_wcs(self):
 
 @pytest.mark.bigdata
 @pytest.mark.xfail
-class TestNIRISSSpec2(BaseJWSTTest):
+class TestNIRSpec2(BaseJWSTTest):
```

Comment:
Theoretically this should be ``TestNIRSpecSpec2`` or just ``TestNRSSpec2``.

Reply:
I like your quibbles.